operator/src/main/java/oracle/kubernetes/operator/calls/unprocessable/UnrecoverableErrorBuilderImpl.java

```diff
@@ -69,6 +69,10 @@ public static boolean isNotFound(ApiException e) {
     return code == 404 || code == 410;
   }
 
+  public static boolean isUnauthorized(ApiException e) {
+    return e.getCode() == 403;
+  }
```

Comment:
Should this also check for 401 error code? I believe 401 is unauthorized and 403 is Forbidden.

Reply:
Good catch. I believe it is only the 403 error that applies here. I think it is better to rename the method to isForbidden.